stable/database/README.md

```diff
@@ -229,6 +229,12 @@ The following tables list the configurable parameters of the `database` chart an
 | `configFilesPath` | Directory path where `configFiles.*` are found | `/etc/nuodb/` |
 | `configFiles.*` | See below. | `{}` |
 | `podAnnotations` | Annotations to pass through to the SM an TE pods | `nil` |
+| `autoImport.*` | Enable and configure the automatic initialising of the initial database state | `disabled` |
+| `autoImport.source` | The source - typically a URL - of the database copy to import | `""` |
+| `autoImport.credentials` | Authentication for the download of `source` in the form `user`:`password` | '""'|
+| `autoImport.stripLevels` | The number of levels to strip off pathnames when unpacking a TAR file of an archive | `1` |
```

Comment:
This assumes that the entire contents of the TAR file are contained in a path of that length. Otherwise it would be ambiguous which path within the TAR file contains the archive. I think it would be better if we took a pathname within the TAR file that contains the archive. Also, it is possible that the archive and journal are in different directories, but I'm not sure how this parameter would account for that.

Reply:
This value of `stripLevels` is transparently passed to `tar` as `--strip-components` option. The typical use case is that the source tarball contains only one archive directory (including the journal).
Currently, an external journal directory is not supported.